snap/local/launch.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/sh
 
 # Make sure a link to desktop launcher is added to the autostart dir
-autostart_dir="$SNAP_USER_DATA/.config/autostart"
-if [ ! -d "$autostart_dir" ]; then
-	mkdir -p "$autostart_dir"
-	ln -sfnt "$autostart_dir/" "$SNAP/share/applications/indicator-sound-switcher.desktop"
+
+AUTOSTART="$SNAP_USER_DATA/.config/autostart/"
+
+if [ ! -d "$AUTOSTART" ]; then
+    mkdir -p "$AUTOSTART"
 fi
```

Comment:
```suggestion
mkdir -p "$AUTOSTART"
```
`mkdir` is a no-op if the directory already exists, so no condition necessary.